x-pack/test_serverless/functional/page_objects/svl_user_profiles.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export function SvlUserProfilePageProvider({ getService }: FtrProviderContext) {
```

Comment:
__question:__ I believe the [`UserProfilePageProvider` service/page-object](https://github.com/elastic/kibana/blob/c49cb1fa9e67c04ed9400b5786311bf4c5bac58d/x-pack/test/functional/page_objects/user_profile_page.ts#L11) is also available in Serverless tests. Do we need some Serverless-specific logic that couldn't fit in that service, so that we need a separate serverless-only user profiles provider?

Reply:
I put there `changeUserProfileTheme` since it's different for serverless, if that's not needed anymore I can move `getProfileEmail` and `getProfileFullname` to `UserProfilePageProvider` and delete this file.